cluster/test-smoke.sh

```diff
@@ -23,10 +23,10 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 
-TEST_ARGS="$@"
+TEST_ARGS="$*"
```

Comment:
Actually, had another look at this.. there's no need to pass this through another variable. We can scrap this line completely, and where TEST_ARGS is referenced below, we can just use `"$@"`

Reply:
Yes, I have updated it